src/Docfx.DataContracts.Common/TocItemViewModel.cs

```diff
@@ -92,6 +92,11 @@ public class TocItemViewModel
     [JsonPropertyName("order")]
     public int? Order { get; set; }
 
+    [YamlMember(Alias = Constants.PropertyName.Type)]
+    [JsonProperty(Constants.PropertyName.Type)]
+    [JsonPropertyName(Constants.PropertyName.Type)]
+    public string Type { get; set; }
```

Comment:
The name `type` is too general for TOC as TOC is language independent unlike in `MetadataItem`. How about `dotnetType`?

Reply:
> language independent 

Does it make sense then that the name is not .NET-related? i.e., we don't include `dotnet` or alike as part of the name

> The name `type` is too general 

Makes sense. Thinking `symbolType` is more descriptive (similar to symbols in Roslyn's semantic model). More accurate than `memberType`, since values like `namespace` are not members.

In any case this might also mean that the `type` that's currently being exposed on inner nodes when using `memberLayout: "separatePages"` should also use `symbolType` instead of `type` for consistency - which might be seen as a breaking change?

I'll explore what changes would be needed to normalize into `symbolType`.

Reply:
So consistently moving from `type` to `symbolType` (or alike) means quite some breaking changes, specially for templates. I'm not a good judge of whether this is a good or a bad idea - but by keeping `type` we can skip / delay the problem altogether (which seems reasonable for backwards compatibility to me).

From what I see `type` is generally used to refer to different concepts e.g., `ApiNames` or `MemberType`, so it's generally ambiguous as it stands now. It's also found referenced from `Constants` and other times as a literal - this is making current maintainability more laborous than needed (I don't have enough context on the reasons why it is currently like this, so take this with a grain of salt).

Reply:
Yep lets keep the name `type` to avoid breaking change. I thought `type` is a new fields in the TOC but it was already exposed.

The changes looks good as proposed. We can merge after publishing the PR and test pass.